Inline policy names for IAM Groups (#621)
inspec/inspec-aws: docs/resources/aws_iam_groups.md

## Properties

|Property              | Description|
| ---                  | --- |
|group\_names          | The group name. |
|group\_ids            | The group ID. |
|arns                  | The Amazon Resource Name of the group. |
|users                 | Array of users associated with the group.  |
|entries               | Provides access to the raw results of the query, which can be treated as an array of hashes. |

> |has\_inline\_policies | Boolean indicating whether or not the Group has policies set directly on it. |

Reviewer: ```suggestion
|has\_inline\_policies | Boolean indicating whether or not the group has policies applied to it. |
```
